jdaviz/configs/cubeviz/plugins/tests/test_parsers.py

```diff
@@ -149,7 +149,7 @@ def test_spectrum3d_no_wcs_parse(cubeviz_helper):
     data = cubeviz_helper.app.data_collection[0]
     flux = data.get_component('flux')
     assert data.label.endswith('[FLUX]')
-    assert data.shape == (3, 2, 4)  # y, x, z
+    assert data.shape == (2, 3, 4)  # y, x, z
```

Comment:
Should the comment change to match the change you've made, or was the comment wrong before?

Reply:
The correctness didn't change, whether it's wrong or right - the size of the x and y axis did change in this PR. How about I just delete the comment 😆

Reply:
Why do I feel like we talked about this before? Didn't you say the comment on L145 was wrong and you were going to fix it?

Reply:
I did, in the test_plugin.py file. I'm just very tired of thinking about what x vs nx are supposed to mean in terms of array vs display order in these comments 😭

Reply:
I find the comments useful though because they tell me what to expect on the display. What if we change it to `n_row`, `n_col`?